.travis.yml

```diff
@@ -1,5 +1,168 @@
-language: haskell
-ghc:
-  - 7.4
-  - 7.6
-  - 7.8
+###############################################################################
```

Comment:
this doesn't look like haskell-ci output. i don't want to block the PR on this, but it should be noted that there has been an effort to migrate to haskell-ci for all core libraries. Again, not a blocker, and I think this is fine for now.

Reply:
I've discussed this with @Bodigrim and I think he might setup haskell-ci, I just never dealt with it personally. I tend to stay in my happy `stack` land ;)